content/docs/how-to-update-your-tina-version-with-dependabot.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: How To Update Your Tina Version With Dependabot
+last_edited: '2025-06-24T15:41:58.484Z'
+next: ''
+previous: ''
+---
+
+If you're using GitHub, Dependabot can automatically keep your TinaCMS packages up-to-date by checking for new versions and opening PRs.
+
+## Configuring Dependabot
+
+You have 2 packages to keep an eye out for:
+
+* `tinacms`
+* `@tinacms/cli`
+
+To do that, just create a dependabot.yml file in your .github folder like this:
+
+```yaml
+# .github/dependabot.yml
+
+version: 2
+updates:
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    allow:
+      - dependency-name: "tinacms"
+      - dependency-name: "@tinacms*"
+```
+
+> Note: You can find more information about the dependabot config here: [Dependabot options reference](https://docs.github.com/en/code-security/dependabot/working-with-dependabot/dependabot-options-reference#about-the-dependabotyml-file)
+
+By doing this, dependabot will create Pull Requests whenever one package is to be updated.
+
+## How to manage the Pull Requests
+
+Now that you have your PRs you have 2 options
+
+### Individual PRs
+
+By default, dependabot opens a separate PR for each package.
+
+As both TinaCMS packages depend on eachother this will lead to conflicts and potential mistakes when merging the PRs one by one.
+
+<Youtube caption="❌ Bad Example - Merging PRs one by one" embedSrc="https://www.youtube.com/embed/iDTZmTAczvA" />
+
+### Grouped PRs (recommended)
```

Comment:
```suggestion
```
not a fan of embedding the YakShave since the video isn't good enough for public